e-commerce-project/src/app/input-page/input-page.component.html

```diff
@@ -0,0 +1,6 @@
+<h3>Welcom to login page!</h3>
+<p>Enter your username and password. Then push Submit button.</p>
+
+<input type="text" name= 'username' placeholder="Username"><br><br>
+<input type="password" name="pass" placeholder="Password"><br><br>
```

Comment:
Please bind data of input tags and create a function to submit ('submit' or 'click' event) and pass input data from both input fields into `input-page.component.ts` component and simply log it into console using console.log: 
1) Implement any way of your choice to bind input tags values for submission:
   A) hash (#name, #pass) identifiers 
   **Note**: docs and examples https://angular.io/guide/user-input#get-user-input-from-a-template-reference-variable
   B) using event object ($event) in submit/click function:
   **Note**: docs and examples https://angular.io/guide/user-input#get-user-input-from-the-event-object
2) create a submit/click function inside `input-page.component.ts` component which accepts and logs into console values of both input fields

Please use any option of your choice.

Reply:
Finally done. [Here](https://github.com/e-commerce-app/e-commerce-ui/blob/story-2/create-a-login-page/e-commerce-project/src/app/input-page/input-page.component.html) and [here](https://github.com/e-commerce-app/e-commerce-ui/blob/story-2/create-a-login-page/e-commerce-project/src/app/input-page/input-page.component.ts) changed files.